src/cunumeric/unary/unary_op_template.inl

```diff
@@ -72,7 +103,7 @@ struct UnaryOpDispatch {
   void operator()(UnaryOpArgs& args) const
   {
     auto dim = std::max(args.in.dim(), 1);
-    double_dispatch(dim, args.in.code(), UnaryOpImpl<KIND, OP_CODE>{}, args);
+    cunumeric::double_dispatch(dim, args.in.code(), UnaryOpImpl<KIND, OP_CODE>{}, args);
```

Comment:
I think you can do better than this by checking the op code from the outside and choosing a point copy functor if the op code was `COPY`. Then, you don't even need to do this custom double dispatch for all the other ops.